classes/db/Db.php

```diff
@@ -871,4 +871,14 @@ public static function ds($sql, $use_cache = 1)
 		Db::s($sql, $use_cache);
 		die();
 	}
+	
+	/**
+	 * Get used link instance
+	 *
+	 * @return PDO|mysqli|resource Resource
+	 */
+	public function getLink()
+	{
+		return $this -> link;
```

Comment:
Hi,

thank you for your contribution, please remove estra space before and after ->

Regards